docs/current/543069-index.md

```diff
@@ -63,6 +63,69 @@ lib -..-> engine -..-> oci -..-> A1 & B1 & C1
 5. When all operations in the pipeline have been resolved, the engine sends the pipeline result back to your program.
 6. Your program may use the pipeline's result as input to new pipelines.
 
+The engine is made of 2 parts: an API router, and a runner.
```

Comment:
I think it would make the lecture more fluent if this new part was in a sub section.
You could have

```
* How does it work?
  * Execution flow
  * Engine
```

Reply:
Thanks for the compliment but actually these diagrams belong to @shykes. I wil modify it for sure!